Automattic/jetpack
# Jetpack: Implement AutoPlay Hovering VPBlock feature

The primary purpose is to explore the resources we have to empower the Video Press block and maybe some suggestions that we would do to keep enhancing the system, to be able to implement the Autoplay hovering feature:

> Add a block or state of the videoplayer where hovering can play a 5 second or so interval.

Fixes #

#### Changes proposed in this Pull Request:

*

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to block editor
* Create a new VideoPress block
* Enable Autoplay feature
* Confirm you can change the starting time by interacting with the Range component
* Enable _Autoplay when Hovering_ feature
* Confirm the video playbacks when hovering over it
* Confirm it stops when leaving the player
* Confirm it stops once the video lapse is achieved (five seconds)
* Confirm that we 🔼 happens, when leaving the player it sets the current position previously defined 



## 🎥 Autoplay / Starting time 
https://user-images.githubusercontent.com/77539/180007980-a256e793-f1d8-440f-905e-c44e23d4cd37.mov

## Autoplay / Hovering
https://user-images.githubusercontent.com/77539/180007959-2020d008-ad5d-4e77-96f0-ac2bb56cdd4e.mov

## Autoplay Hovering / Front-end
https://user-images.githubusercontent.com/77539/180007775-b8a6b017-f007-4284-984c-84b57c4aa78d.mov
